app/models/concerns/wp_cache.rb

```diff
@@ -73,11 +101,18 @@ def purge(wp_id)
     # Convenience method for calling the WP API.
     #
     # TODO (cies): re-raise any connection errors with more intuitive names
-    def get_from_wp_api(route)
-      response = Faraday.get "#{ Rails.configuration.x.wordpress_url }?json_route=/#{ route }"
+    def get_from_wp_api(route, page = -1)
+      # TODO (dunyakirkali) pass filter through args to get_from_wp_api
+      posts_per_page = (ENV['PER_PAGE'].to_i == 0 ? -1 : ENV['PER_PAGE'].to_i)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.